platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationFileInstallerTest.java

```diff
@@ -81,10 +82,11 @@ public void testInstall() throws Exception {
         File testFile = new File(File.class.getResource(TEST_FILE_NAME)
                 .getPath());
 
-        assertTrue("Returned URI should have a unix-style path",
-                testInstaller.install(testFile)
-                        .getPath()
-                        .matches("(/.+)+"));
+        URI mUri = ApplicationFileInstaller.install(testFile);
```

Comment:
Don't worry - this was done awhile back (when I first arrived at Connexta). I've kicked my Hungarian notation habit thanks to @kcwire and my IDE. It's fixed.